pylace/lace/utils.py

```diff
@@ -117,3 +122,46 @@ def infer_column_metadata(
         ).rename(column)
         mds.append(md)
     return mds
+
+
+_COMMON_TRANSITIONS = {
+    "sams": [
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.component_parameters(),
+        StateTransition.row_assignment(RowKernel.slice()),
+        StateTransition.component_parameters(),
+        StateTransition.column_assignment(ColumnKernel.gibbs()),
+        StateTransition.column_assignment(ColumnKernel.slice()),
+        StateTransition.view_alphas(),
+        StateTransition.feature_priors(),
+    ],
+    "flat": [
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.row_assignment(RowKernel.sams()),
+        StateTransition.view_alphas(),
+        StateTransition.component_parameters(),
+        StateTransition.row_assignment(RowKernel.slice()),
+        StateTransition.component_parameters(),
+        StateTransition.view_alphas(),
+        StateTransition.feature_priors(),
+    ],
+    "fast": [
+        StateTransition.row_assignment(RowKernel.slice()),
+        StateTransition.column_assignment(ColumnKernel.slice()),
+    ],
+}
+
+
+def _get_common_transitions(name: str) -> List[StateTransition]:
+    transitions = _COMMON_TRANSITIONS.get(name, None)
+    if transitions is None:
+        raise ValueError(
+            f"{name} is not a valid transitions set name. valid options are..."
```

Comment:
can you print out the common transition keys here instead of `...`

Reply:
Don't know why I didn't see that. Good catch.